src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/PatientMapper.java

```diff
@@ -78,10 +78,11 @@ private void mapNhsNumberIdentifier(final String nhsNumber, final Patient patien
             .setValue(nhsNumber)
             .addExtension()
             .setUrl("https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-NHSNumberVerificationStatus-1")
-            .setValue(new CodeableConcept().addCoding(new Coding()
-                .setSystem("https://fhir.hl7.org.uk/STU3/CodeSystem/CareConnect-NHSNumberVerificationStatus-1")
-                .setCode("01") //TODO awaiting NHS IOPS to provide answer
-                .setDisplay("Number present and verified")
+            .setValue(new CodeableConcept().addCoding(
```

Comment:
Just noticed above we set the value to the `nhsNumber`, and then set it again here. I assume the first `setValue` isn't needed if we're overwriting it?

Reply:
Nevermind, just noticed this is an extension